### DISCUSSION TITLE
Hi. What is the license for this open source?

### DISCUSSION THREAD

**Discussion by Ufarufa** [visioncortex/vtracer#17] [Q&A] [answered]
I like this project(vtracer)
I want to use Vtracer in my company's product, but I don't know the license, so I have a question.

**Comment by tyt2y3** [comment]
It's in the respective crate's sub-directory.

**Comment by grahammiln** [comment, marked as answer]
MIT or Apache, see https://github.com/visioncortex/vtracer/tree/master/cmdapp and https://crates.io/crates/vtracer
